packages/components/segmented/src/types.ts

```diff
@@ -0,0 +1,11 @@
+export type Option =
+  | {
+      label: string
+      value: string | number | boolean
+      disabled?: boolean
+      [key: string]: any
+    }
+  | string
+  | number
+  | boolean
+  | undefined
```

Comment:
`Option` Support` undefiend`? What situation needs this support? 🤔

Reply:
In order for ts to compile successfully, you can remove undefined and try building it locally